### DISCUSSION TITLE
زمان بندی

### DISCUSSION THREAD

**Discussion by Ali7268** [amir1376/ab-download-manager#187] [Ideas]
سلام بسیار عالی درست کردید خیلی خیلی حالم کردم فقط یه قسمت زمان بندی دانلود(منظورم این هست که مثل دانلود منیجر زمان بهش بدی خودش دانلود کنه و بعد از دانلود خاموش بشه ) هم اضافه کنید تکمیل تکمیل میشه ممنون

**Comment by amir1376** [comment]
درود بر شما. این ویژگی الان هم وجود داره در بخش صف ها میتونید فعالش کنید

**Reply by ebrahim-zarei** [reply]
ضمن تشکر امکان خاموش کردن هم اضافه بشه عالیه
